src/app/components/Curation/HierarchicalGrid/index.tsx

```diff
@@ -140,9 +140,13 @@ const HiearchicalGrid = ({
                 <Promo.Body className="promo-paragraph" css={styles.body}>
                   {promo.description}
                 </Promo.Body>
-                <Promo.Timestamp className="promo-timestamp">
-                  {promo.lastPublished}
-                </Promo.Timestamp>
+                {!isLive ? (
+                  <Promo.Timestamp className="promo-timestamp">
+                    {promo.lastPublished}
+                  </Promo.Timestamp>
+                ) : (
+                  <div />
```

Comment:
This isn't particularly tidy as we're returning an empty div we aren't using. We tried returning `null` but that doesn't appear to be an option as React expects a valid element and the page won't render (`internal server error`). Returning an empty string `''` results in TS errors despite compiling which we could suppress as an alternative.

Is there another approach which could be suggested?

 Can we keep this change temporarily until the new implementation is added where we add the more accurate timestamp to the promo down the line?

Reply:
Update: We tried adding a React `fragment` yesterday but we were missing an import. Adding this instead is arguably an improvement as it returns a React element but not a `div`.

Reply:
`null` should be a valid return in React, we have used it in other areas, for example: https://github.com/bbc/simorgh/blob/d03d06e0a0e3025b46c1546d297a3f4da93ff0e9/src/app/legacy/containers/CpsRecommendations/index.jsx#L105-L119

Reply:
Hmm It might be to do with the way these promos are implemented but when we attempt to return `null`, including with a server reboot, the page fails to render. Might have to unpick some of the logic to determine why that's happening hence the `Fragment`

Reply:
Now returning `null` 😄